Add Top section solves #56

kubami9 opened the pull request:
### Add top section similar to most FOSSASIA websites. Fix #56 

**This PR includes:**
 - full width header with FOSSASIA background-image (the same which is on FOSSASIA Summit 2012 and 2018)
 - partners/sponsors section which includes all content which has been in the sidebar, apart from the Flickr images
 - delete sidebar
 - FOSSASIA newsletter section (basically email subscription from sidebar turned into own section, somewhat similar to this from fossasia.org)
 - center content
 - menu background changed to transparent and font color to white
 - some simple bug and typo fixes

Preview link: https://kubami9.github.io/2014.fossasia.org-km-contribute/

isuruAb reviewed (changes requested):
Hello @kubami9  there are some changes to be done.
logo is missing and font size should be little bit smaller. please follow the styles of fossasia.org and don't forget to squash your commits.
cheers

kubami9 commented:
@isuruAb done 😄

codedsun commented on _includes/header.html:
```diff
@@ -1,52 +1,61 @@
+		h1 {
+			font-family: 'Open Sans';
+			font-weight: normal;
+      		text-transform: uppercase;
```
The indentation needs to be corrected

kubami9 commented:
@isuruAb could you check out my changes 😄 this PR is getting more and more conflicts to resolve and whenever I resolve them I need to squash and it all loops

akshatnitd reviewed (commented):
The changes looks good. You just need to fix the indentation issues at various lines, keeping in view the best practices. And, lastly squash the commits! The, it would be good to go :+1:

kubami9 committed:
Add top section

Add top section and delete side-bar

Simple typo fix

Simple typo fix

Add FOSSASIA logo in menu and change font size

Add FOSSASIA logo in menu and change font size to fit fossasia.org

Indentation correct

Indentation correct

Responsiveness improved, social icons moved below the side-bar and side-bar turned to a flex object.   (#72)

* Set theme jekyll-theme-cayman

* Set theme jekyll-theme-minimal

* Update _config.yml

*  Issue#2 solved by changing padding dimensions from pixels  to percentages

* solved issue#2 better by aligning h1 text to the centre and removing padding

* solved issue#1 by changing the height #content

* fixed issue#1 by changing hjeight of #content to 100%

* fixed issue#1 better by changing the height for #wrapper to auto

* changes the font sizes of different menu buttons to suitable percentages so that the menu does not break on any resolution, issue #3 resolved

* more font sizes changes

* firxed side-bar by turning theheight value to auto

* moved social links to side  bar and changed side bar to flex object

* issues regarding styles solved by committing a previous version

* aa

* path changed

* final update

Partners section bug fix

Partners section bug fix

Partner sercion bug fixes

Some content was missing

Resolving conflicts

Resolving conflicts

Fix indentation issue with header.html

Fix indentation issue with header.html
Fix indentation issue with top.html

Fix indentation issue with top.html

Responsiveness improved, social icons moved below the side-bar and side-bar turned to a flex object.   (#72)

* Set theme jekyll-theme-cayman

* Set theme jekyll-theme-minimal

* Update _config.yml

*  Issue#2 solved by changing padding dimensions from pixels  to percentages

* solved issue#2 better by aligning h1 text to the centre and removing padding

* solved issue#1 by changing the height #content

* fixed issue#1 by changing hjeight of #content to 100%

* fixed issue#1 better by changing the height for #wrapper to auto

* changes the font sizes of different menu buttons to suitable percentages so that the menu does not break on any resolution, issue #3 resolved

* more font sizes changes

* firxed side-bar by turning theheight value to auto

* moved social links to side  bar and changed side bar to flex object

* issues regarding styles solved by committing a previous version

* aa

* path changed

* final update

Indentation fixes

Indentation fixes

Responsiveness improved, social icons moved below the side-bar and side-bar turned to a flex object.   (#72)

* Set theme jekyll-theme-cayman

* Set theme jekyll-theme-minimal

* Update _config.yml

*  Issue#2 solved by changing padding dimensions from pixels  to percentages

* solved issue#2 better by aligning h1 text to the centre and removing padding

* solved issue#1 by changing the height #content

* fixed issue#1 by changing hjeight of #content to 100%

* fixed issue#1 better by changing the height for #wrapper to auto

* changes the font sizes of different menu buttons to suitable percentages so that the menu does not break on any resolution, issue #3 resolved

* more font sizes changes

* firxed side-bar by turning theheight value to auto

* moved social links to side  bar and changed side bar to flex object

* issues regarding styles solved by committing a previous version

* aa

* path changed

* final update

Fix merge error

Fix merge error

kubami9 commented:
@akshatnitd Is it good now?

nzec reviewed (approved):
There are a lot of problems but I think they can be improved over time with different issues. 

I think this should be merged first.

@kubami9 Awesome work! 👍

akshatnitd commented on _includes/header.html:
```diff
@@ -389,20 +369,117 @@
-
 
+  .top {
+  background-image: url(images/fossasia2016.jpg);
```
Missing indentation here and same in many lines below. Also, I can see 2 commits, make sure only 1 commit remains after squashing.

nzec commented on _includes/header.html:
```diff
@@ -208,14 +180,21 @@
+}
 /*Strip the ul of padding and list styling*/
 ul {
+<<<<<<< HEAD
```
Fix merge errors.

kubami9 commented:
It's confusing, whenever I squash commits it gets conflicts and whenever I resolve conflicts it needs to be squashed

kubami9 commented:
And now - I don't even know why - it shows me that I changed 93 files

kubami9 committed:
Resolve conflicts

kubami9 commented:
@akshatnitd  Now it should be okay, however I can't merge it into just one commit since it's giving dozens of conflicts

akshatnitd reviewed (approved):
Good work! :) Merging :rocket: 
Further improvements can be done by next PRs.

kubami9 commented:
Yaay! Thank you so much @akshatnitd!!! 😃
